### DISCUSSION TITLE
Pokemon-Pythonred organisation membership

### DISCUSSION THREAD

**Discussion by nasserkessas** [Pokemon-PythonRed/Pokemon-PythonRed#87] [Q&A] [answered]
Hello @TurnipGuy30, I was just wondering what requirements you have in order to join the GitHub Pokemon-Pythonred organisation if it's possible for outsiders to do so. If you want to keep the membership small or don't want anyone else to join, that's ok too, just curious.

P.S. I don't actually know too much about how organisations work so if I am asking for something that's not possible or something please let me know.

**Comment by riverpaper** [comment, marked as answer]
If you're saying that you'd like to join, I'd be happy to add you! Thanks again for all your help, I'll invite you tomorrow.

**Reply by nasserkessas** [reply]
Yes, I'd love to, thanks a lot. I would love to help out more in the future.

Also: I'll mark this as the answer you get credit

**Reply by riverpaper** [reply]
You've now been invited!

**Reply by nasserkessas** [reply]
Thanks!!!
